index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Static Web Quiz</title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+<h1> Static Web Quiz</h1>
+
+  <article class="article">
+    <h1 class="cardTitle"> Red Cards </h1>
+      
```

Comment:
Be aware of whitespace left in your files. Inevitably it will cause headaches in the form of merge conflicts. The best way to stem it is to either download the trailing spaces package for sublime to highlight your whitespace or add a line in your user config preferences to "trim_trailing_white_space_on_save": true.